app/src/sharedTest/java/org/oppia/app/profile/AddProfileChooserActivityTest.kt

```diff
@@ -59,7 +59,7 @@ import javax.inject.Singleton
 
 @RunWith(AndroidJUnit4::class)
 @LooperMode(LooperMode.Mode.PAUSED)
-class AddProfileActivityTest {
+class AddProfileChooserActivityTest {
```

Comment:
Update all test cases names too.

Reply:
@rt4914 I need to revert all the changes in the file?

Reply:
Yes, I missed this in this PR but I have commented accordingly in other PR.